public/app/features/alerting/unified/components/rules/Filter/RulesFilter.v1.tsx

```diff
@@ -150,7 +152,10 @@ const RulesFilter = ({ onClear = () => undefined }: RulesFilerProps) => {
     trackRulesSearchComponentInteraction('contactPoint');
   };
 
-  const canRenderContactPointSelector = config.featureToggles.alertingSimplifiedRouting ?? false;
+  const canRenderContactPointSelector =
```

Comment:
Otherwise we would render the filter selector when user potentially couldn't see it

Reply:
This is an interesting edge case. If I understand correctly, if the user has no `AlertingReceiversRead` permission we cannot use simplify routing and we need to default to notification policies?

Reply:
Yes - but also I've been thinking about edge cases of them being able to see contact points, but not necessarily the contact point associated with an alert rule they can see. So perhaps in that case we should disable the simplified routing selector?